### DISCUSSION TITLE
What is the best practice to create a bundle?

### DISCUSSION THREAD

**Discussion by carstep** [symfony/symfony#42599] [Q&A] [answered]
Dear Community I asked this, because I am confused about the documentation / recommendation and real example for this subject:
I researched this sites for information: 
- https://symfony.com/doc/current/bundles/best_practices.html
- https://symfony.com/doc/current/bundles.html
- some insights from real examples like framework-bundle and twig-bundle
they not seem to be on the same level. How to go on to create my first bundle, which reference should I follow?

**Comment by faizanakram99** [comment, marked as answer]
First one https://symfony.com/doc/current/bundles/best_practices.html 
(Second link doesn't show new bundle structure, it is still using the old dir structure)
